pkgs/tools/security/tcb/default.nix

```diff
@@ -48,7 +48,7 @@ stdenv.mkDerivation rec {
     '';
     homepage = "https://www.openwall.com/tcb/";
     license = licenses.bsd3;
-    platforms = platforms.linux;
+    platforms = systems.inspect.patterns.isGnu;
```

Comment:
Heads up: this broke https://search.nixos.org. I made a PR with a quick fix: https://github.com/NixOS/nixos-search/pull/602. Eventually we should figure out how to display patterns properly.

Reply:
Also breaks nixpkgs-review after #213780.